src/bin/clean_up.js

```diff
@@ -0,0 +1,217 @@
+                        deletions.push(storage.deleteProject({
+                            projectId: result.projectId,
+                            username: params.username
+                        })
+                            .then(function (removed) {
+                                return {
+                                    removed: removed,
+                                    projectId: result.projectId
+                                };
+                            })
+                            .catch(function (err) {
+                                return {
+                                    removed: false,
+                                    projectId: result.projectId,
+                                    err: err
+                                };
+                            })
+                        );
+                    }
+                } else {
+                    keep += 1;
+
+                }
+            });
+
+            logger.info('Will keep', keep, 'projects.');
+
+            return Q.all(deletions);
+        })
+        .then(function (results) {
+            var cnt = 0;
+            results.forEach(function (res) {
+                if (res.removed === true) {
+                    cnt += 1;
+                    logger.info('Removed', res.projectId);
+                } else {
+                    logger.error('Failed to remove ' + res.projectId, res.err.stack);
+                }
+            });
+
+            logger.info('Removed', cnt, 'projects!');
+        })
+        .catch(function (err_) {
+            err = err_;
+        })
+        .finally(function () {
+            logger.debug('Closing database connections...');
+            return Q.allSettled([storage.closeDatabase(), gmeAuth.unload()])
+                .finally(function () {
+                    logger.debug('Closed.');
+                    if (err) {
+                        throw err;
+                    }
+                });
+        });
+}
+
+module.exports = cleanUp;
+
+if (require.main === module) {
+    var Command = require('commander').Command,
+        program = new Command();
+
+    program
+        .version('1.6.0')
+        .option('-d, --del [boolean]', 'If true will do the deletion [false].', false)
+        .option('-l, --list [boolean]', 'If true will list all the projects this user has access to [false].', false)
+        .option('-t, --daysAgo [number]', 'Minimum age (last viewed) of a project to delete [10].', 10)
+        .option('-c, --commits [number]', 'Maximum number of commits of a project to delete [1].', 1)
+        .option('-b, --branches [number]', 'Maximum number of branches of a project to delete [1].', 1)
+        .option('-r, --regex [string]', 'Project names must match the regexp [.*].', '.*')
+        .option('-u, --username [string]', 'the user that will remove the projects [guest account]')
+        .on('--help', function () {
+            console.log('  Examples:');
```

Comment:
For the examples, I suggest to use the long version of the commands, for instance:

    node clean_up.js --list